package.json

```diff
@@ -1,15 +1,28 @@
 {
   "name": "salesforcedx-vscode-mobile",
-  "displayName": "Salesforce Mobile Extensions for Visual Studio Code",
+  "displayName": "%extension.displayName%",
   "description": "",
   "version": "0.0.1",
-  "publisher": "salesforce",
+  "publisher": "Salesforce",
```

Comment:
This was correct the original way. Cf. https://github.com/forcedotcom/salesforcedx-vscode/blob/c8c2b544703d8b1b2f5dc072d6c289aa3e6c51dd/packages/salesforcedx-vscode-core/package.json#L19

Reply:
If I dont change it, it shows up lower case in the extensions, which is why I changed it, but maybe there is another way. Will investigate:

<img width="527" alt="image" src="https://github.com/salesforce/salesforce-offline-vscode/assets/2682021/6abc75bc-f1fa-4807-8b02-4089becbc8aa">

Reply:
You can see in the URL (e.g. https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) that the name is lowercase, so there must be something else that's governing the display of that name.